antenna-documentation/pom.xml

```diff
@@ -71,6 +72,39 @@
                 <skip.tests>false</skip.tests>
             </properties>
         </profile>
+        <profile>
+            <id>site-deploy</id>
+            <activation>
+                <property>
+                    <name>site-deploy</name>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>com.github.github</groupId>
+                        <artifactId>site-maven-plugin</artifactId>
+                        <configuration>
+                            <message>Creating documentation on Github Pages for ${project.name} ${project.version}</message>
+                            <merge>true</merge>
+                            <path>${project.version}</path>
+                            <noJekyll>true</noJekyll>
+                            <userName>${GITHUB_USERNAME}</userName>
```

Comment:
one theoretically would have to xml-escape these variables, since they could contain `<`. Or is this no problem with env substitution in maven poms?